src/parquet/util/test-common.h

```diff
@@ -104,8 +115,72 @@ void random_bytes(int n, uint32_t seed, std::vector<uint8_t>* out) {
   }
 }
 
-} // namespace test
+template <typename T>
+void random_int_numbers(int n, uint32_t seed, std::vector<T>* out) {
+  std::mt19937 gen(seed);
+  std::uniform_int_distribution<T> d(-10000, 10000);
```

Comment:
The behavior of this template is not defined for `Int96` if I understand the C++11 documentation correctly. I think we're going to need a separate data generator for Int96 that generates 3 random `uint32_t` values